### DISCUSSION TITLE
修改css样式后本地正常显示但再次提交后vercel上现实的页面无变化

### DISCUSSION THREAD

**Discussion by ZHANQILU** [Candinya/Kratos-Rebirth#148] [Q&A]
如题，我已经将详细情况和附图发到大佬你的邮箱了，但你没回复，想着你可能没看到

**Comment by ArcticLampyrid** [comment]
考虑可能是vercel的cdn节点的缓存或者是你浏览器的缓存造成的


Sent from Outlook for Android<https://aka.ms/ghei36>
________________________________
From: ZHANQILU ***@***.***>
Sent: Wednesday, August 24, 2022 4:38:00 PM
To: Candinya/Kratos-Rebirth ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Candinya/Kratos-Rebirth] 修改css样式后本地正常显示但再次提交后vercel上现实的页面无变化 (Discussion #148)


如题，我已经将详细情况和附图发到大佬你的邮箱了，但你没回复，想着你可能没看到

―

**Reply by ZHANQILU** [reply]
节点刷新了，浏览器缓存也清理了，还是没用，页面还是没改变

**Comment by ArcticLampyrid** [comment]
请提供相关网址

Sent from Outlook for Android<https://aka.ms/ghei36>
________________________________
